docker/build/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:stable
```

Comment:
It may be preferable to depend on a fixed stable release. Would `debian:jessie` work?